### ISSUE TITLE
Web Application

### ISSUE THREAD

**Issue by GoelJatin** [HydPy/HydPy#33]
#### Few items to be included:

1. Add a page for Users to request sessions.
1. Add a page for users to provide their mail address, and register for upcoming events.
1. Add a page for Organisations to list their offices as venues for events
1. Add a page for Speakers to provide information on a meetup / workshop they wish to do
1. Integrate the Meetup group, Facebook page, and the twitter handle with the application to automatically publish any information / update on the upcoming events / anything related to the community.
1. Search for all speakers / talks available.

More could be added later.

I'm thinking of writing a complete Django app for this, and have started on the design.

Please label it as Hacktoberfest.

**Comment by GoelJatin** [comment]
Source Repo here: https://github.com/GoelJatin/HydPy

**Comment by GoelJatin** [comment]
@ananyo2012 

Oh yeah, that is already in the plan, but missed to add it in the description.

The App is going to be integrated with all our Social Media platforms and automated, so as soon as an event is created, Meetup will also be created for the same automatically, and the details of the event along with the URLs for Event and Meetup will be shared on all SM platforms.


The App will be integrated with GitHub for Registrations.

**Comment by chirag200666** [comment]
The idea is good and such a project can be used by some of the other communities as well.

**Comment by GoelJatin** [comment]
Thanks @chirag200666 

Got busy with some other stuff, so wasn't able to focus on this in a while.

Will create and share the design doc soon.

**Comment by bhansa** [comment]
We should make it an opensource project under HydPy. In that way we will have our first opensource project for the community by HydPy and it will attract people and communities to use or contribute.
We can also call for contribution during any devsprints etc.

@ananyo2012 @inovizz @chirag200666 thoughts.

**Comment by inovizz** [comment]
Yeah, I had two things in my mind when I started discussion around it. First we'll be doing a Hacktoberfest event in the third week of October I think and secondly, this can be a good opportunity for PyConfHyd volunteers to contribute as well.

**Comment by ananyo2012** [comment]
Sounds like a plan. Let's get this kicking.

**Comment by inovizz** [comment]
yeah, most of us would be there in Chennai for PyConIndia - so its a good opportunity for us to sit together and brainstorm before outlining the milestones.
